DecipherNow/gm-fabric-dashboard
# Update dependencies

## Proposed Changes

  - Update dependencies
  - Modify code based on migration guides
  - Add Prettier badge to README
  
## How to Test

  - npm test and confirm everything passes
  - Run the app and confirm that things work
  - Run storybook and confirm that things work
